orchestra/proc-macro/src/parse/parse_orchestra_struct.rs

```diff
@@ -594,6 +648,34 @@ impl OrchestraGuts {
 						(attr_tokens, span)
 					})
 				});
+
+			let mut cfg_attr =
+				attrs.iter().filter(|attr| attr.style == AttrStyle::Outer).filter_map(|attr| {
+					let span = attr.path.span();
+					attr.path.get_ident().filter(|ident| *ident == "cfg").map(move |_ident| {
+						let attr_tokens = attr.tokens.clone();
+
+						(attr_tokens, span)
+					})
+				});
+
+			let feature_gates = if let Some((attr_tokens, span)) = cfg_attr.next() {
+				// multiple `#[cfg(..)]` found
+				if let Some((_attr_tokens2, span2)) = cfg_attr.next() {
+					return Err({
+						let mut err = Error::new(span, "The first cfg annotation is at");
+						err.combine(Error::new(span2, "but another here for the same field."));
+						err
+					})
+				}
```

Comment:
Shouldn't this just recombine as an `and`?

Reply:
Absolutely correct! Done!